tools/libaqr/images.py

```diff
@@ -108,7 +108,7 @@ def _check_for_image(cls, path: Path) -> Tuple[str, Path]:
             elif imgpath.suffix == ".box" and entry.count("vagrant") > 0:
                 img = imgpath
 
-        if not raw_img or not img:
+        if not raw_img and not img:
```

Comment:
This will not raise if `img` is missing, as if it's everything just fine. I think the bigger question is: when is it possible for .box to be present, but not .raw? If there's a reasonable expectation that it might be the case without it being an error somewhere else, then the check should check for `not img` standalone.

TL;DR: this patch introduces a regression.

Reply:
Gonna stash it, as for when it is possible .raw not present, then this is exactly the case when we run it on Ubuntu.